apps/mealplanr/src/components/recipes/Recipes.tsx

```diff
@@ -116,44 +82,51 @@ function handleNextImage(e: React.SyntheticEvent<HTMLImageElement, Event>) {
 
 interface RecipeProps {
 	type: 'wide'|'tall',
-	Id: number,
+	recipeId: string,
 	At?: string,
 	personal: boolean,
 }
 
 export function Recipe(props: RecipeProps) {
-	const {type, Id, At, personal} = {At: null, ...props};
-	const {Images, Title, Decs, Time, Rating, Ratings, category} = recipeInfo(Id);
+	const {type, recipeId, At, personal} = {At: null, ...props};
+	const [crecipe, setCrecipe] = useState<RecipesType>({
```

Comment:
I don't get why it is called `crecipe` or `setCrecipe`

Reply:
c for current, to differentiate between recipe and Recipe, well naming changed for the better